### ISSUE TITLE
Storage adapter customization

### ISSUE THREAD

**Issue by kjetilge** [perak/kitchen-site#232]
I can't fin a way add stores to an FS.Collection.
It would be nice to have an extra field in "collection" where one could specify a file with an array of extra stores. (in addition to files)

Like:
´´´
//extra_stores.js
[
        new FS.Store.FileSystem("thumbs", {
                transformWrite: function(fileObj, readStream, writeStream) {
                 gm(readStream, fileObj.name()).resize('200', '200').stream().pipe(writeStream);
        },
        new FS.Store.FileSystem("avatar", {
                transformWrite: function(fileObj, readStream, writeStream) {
                 gm(readStream, fileObj.name()).resize('80', '80').stream().pipe(writeStream);
        }
      })
]
´´´

I've tried to find a way to fake it, but right now all I can do is to add a second files.js file with the complete Collection definition and remove the generated one from the output folder.

Is there a better way to get transforms right now ?

**Comment by ljack** [comment]
In situations like this (e.g something required not possible by the cli ) diff and patch has solved the problems. Old goog makefile / grunt / npm / gulp / sh-script with proper invocations works like a treat ;)

Simple, but over complicated build script for one of my projects:
```
#!/bin/bash

ID=insert_app_id_here
URL=http://www.meteorkitchen.com/api/getapp/json/$ID

function build {
 echo Starting meteor-kitchen build
 meteor-kitchen pitpa.json OUTPUT
 # here's the patch to support google login with additional scopes
 patch -p0 < login.patch
}

# fetch the json and also make a backup 
function once {
    curl -s  $URL > pitpa.json.new
    mv pitpa.json.new pitpa.json
    cp pitpa.json backup/pitpa.json-"`date`"
    # git add pitpa.json
    # git commit -m "Autobuild running for $ID"
    build 
}

# run the build once ;)
once

```

**Comment by tripflex** [comment]
Yeah i'm interested in this as well, mainly to be able to create thumbnails to use as needed:

https://github.com/CollectionFS/Meteor-CollectionFS#image-manipulation
